chat-components/src/common/Constants.ts

```diff
@@ -51,4 +51,152 @@ export enum ElementType {
+    public static readonly Icon = "icon";
+    public static readonly Text = "text";
+};
+
+export const IconNames = class {
+    public static readonly DeclineCall = "DeclineCall";
+    public static readonly Microphone = "Microphone";
+    public static readonly MicOff2 = "MicOff2";
+    public static readonly Video = "Video";
+    public static readonly VideoOff = "VideoOff";
+    public static readonly IncomingCall = "IncomingCall";
+    public static readonly ChromeClose = "ChromeClose";
+    public static readonly Download = "Download";
+    public static readonly Mail = "Mail";
+    public static readonly Volume3 = "Volume3";
+    public static readonly Volume0 = "Volume0";
+    public static readonly ChromeMinimize = "ChromeMinimize";
+};
+
+export const AriaLabels = class {
+    public static readonly EndCall = "End Call";
+    public static readonly MicMute = "Mute";
+    public static readonly MicUnmute = "Unmute";
+    public static readonly VideoTurnCameraOn = "Turn camera on";
+    public static readonly VideoTurnCameraOff = "Turn camera off";
+    public static readonly IncomingCallArea = "Incoming call area";
+    public static readonly RejectCall = "Reject call";
+    public static readonly AcceptVoiceCall = "Accept voice call";
+    public static readonly AcceptVideoCall = "Accept video Call";
+    public static readonly DeclineCall = "Decline Call";
+    public static readonly AudioCall = "Audio Call";
+    public static readonly VideoCall = "Video Call";
+    public static readonly LetsChatWeAreOnline = "Let's chat we are online";
+    public static readonly UnreadMessageString = "you have new messages";
+    public static readonly Close = "Close";
+    public static readonly CloseChat = "Close Chat";
+    public static readonly CancelReturnToChat = "Cancel. Return to Chat";
+    public static readonly DownloadChatTranscript = "Download chat transcript";
+    public static readonly EmailTranscript = "Email Transcript";
+    public static readonly TurnSoundOff = "Turn sound off";
+    public static readonly TurnSoundOn = "Turn sound on";
+    public static readonly Minimize = "Minimize";
+    public static readonly EmailChatTranscriptPane = "Email Chat Transcript Pane";
+    public static readonly InputValidationPaneInput = "Email this chat transcript. This will be sent after your chat ends. Email address text area";
+    public static readonly Send = "Send";
+    public static readonly Cancel = "Cancel";
+    public static readonly ProactiveChatPane = "Proactive Chat Pane";
+    public static readonly ChatNow = "Chat Now";
+    public static readonly ReconnectChatPane = "Reconnect Chat Pane";
+    public static readonly ReconnectChatPaneIcon = "Reconnect Chat Pane Icon";
+    public static readonly ContinueConversation = "Continue conversation";
+    public static readonly StartNewConversation = "Start new conversation";
+};
+
+export const EventNames = class {
+    public static readonly onClick = "onClick";
```

Comment:
If using PascalCase for the variable names elsewhere, use it here too

Reply:
fixed